src/app/Plans/components/PlanActionsDropdown.tsx

```diff
@@ -30,28 +31,46 @@ const PlansActionsDropdown: React.FunctionComponent<IPlansActionDropdownProps> =
         isOpen={kebabIsOpen}
         isPlain
         dropdownItems={[
-          <DropdownItem
-            isDisabled={!!plan.status?.migration?.started}
-            onClick={() => {
-              setKebabIsOpen(false);
-              history.push(`/plans/${plan.metadata.name}/edit`);
-            }}
-            key="Edit"
+          <ConditionalTooltip
+            key="edit"
+            isTooltipEnabled={!!plan.status?.migration?.started}
+            content={'This plan cannot be edited because it has been started'}
           >
-            Edit
-          </DropdownItem>,
-          <DropdownItem
-            isDisabled={
+            <DropdownItem
+              isDisabled={!!plan.status?.migration?.started}
+              onClick={() => {
+                setKebabIsOpen(false);
+                history.push(`/plans/${plan.metadata.name}/edit`);
+              }}
+            >
+              Edit
+            </DropdownItem>
+          </ConditionalTooltip>,
+          <ConditionalTooltip
+            key="Delete"
+            isTooltipEnabled={
               hasCondition(conditions, PlanStatusAPIType.Executing) || deletePlanResult.isLoading
             }
-            onClick={() => {
-              setKebabIsOpen(false);
-              toggleDeleteModal();
-            }}
-            key="Delete"
+            content={
+              hasCondition(conditions, PlanStatusAPIType.Executing)
+                ? "This plan cannot be deleted because it's running"
```

Comment:
@gildub This is ready to merge as soon as we replace `it's` with `it is` in these two strings.

Reply:
Done